model-optimizer/mo/ops/convolution.py

```diff
@@ -237,7 +236,7 @@ def infer(node: Node):
         for n in node.out_nodes():
             node.out_node(n).shape = output_shape
 
-        mark_input_bins(node, start_port=1 if node.type != 'DeformableConvolution' else 2)
+        #mark_input_bins(node, start_port=1 if node.type != 'DeformableConvolution' else 2)
```

Comment:
why is this commented out? It can affect `Convolution` operation.

Reply:
could you explain why we should mark some inputs as bin?

Reply:
I propose to stay `mark_input_bins` because we have multiple transformations that use `bin` attribute. Or to remove it at all and arrange DL benchmark validation to see any regressions.

Reply:
reverted